Public/New-PolarisStaticRoute.ps1

```diff
@@ -57,19 +59,21 @@ function New-PolarisStaticRoute {
     if ( -not ( Test-Path -Path $FolderPath ) ) {
         Write-Error -Exception FileNotFoundException -Message "Folder does not exist at path $FolderPath"
     }
-
-    $NewDrive = (New-PSDrive -Name "PolarisStaticFileServer$([guid]::NewGuid().guid)" `
-            -PSProvider FileSystem `
-            -Root $FolderPath `
-            -Scope Global).Name
+    
+        $NewDrive = (New-PSDrive -Name "PolarisStaticFileServer$([guid]::NewGuid().guid)" `
+                -PSProvider FileSystem `
+                -Root $FolderPath `
+                -Scope Global).Name
     
     $Scriptblock = {
         $Content = ""
 
         $LocalPath = $Request.Parameters.FilePath
```

Comment:
If I'm reading this code right I think we could save a lot of code by inserting a bit of logic up here like this to just change the $LocalPath variable to be one of the default HTML files if the user doesn't request a specific one (i.e. hits the root).

```suggestion
        $LocalPath = $Request.Parameters.FilePath
        if(-not $LocalPath) {
            foreach($FileName in $StandardHTMLFiles){
                $FilePath = Join-Path "$($NewDrive):" -ChildPath "$FileName"
                if(Test-Path -Path $FilePath){
                    $LocalPath = $FileName
                    break
                }
            }
        }
```
Then we can avoid adding any new routes and also reuse all of the file serve logic.